### ISSUE TITLE
Can't reference values from section which was included from another file

### ISSUE THREAD

**Issue by Probotect0r** [jaagr/polybar#776]
I have this in my config:
```
[colors]
include-file = /home/myUser/.config/polybar/colors

[bar/default]
background = ${colors.background}
foreground = ${colors.foreground}
```
In the included file I have:
```
background = #2b2c33 
foreground = #d9c1a9
blk = #303139
red = #b16c7f
grn = #92ab75
ylw = #d9c1a9
blu = #485476
mag = #746081
cyn = #636f7d
wht = #d9c1a9
```
I am getting the error `Unexisting reference defined [colors.background]`.
Is it possible to reference values from sections that were included from another file?

**Comment by patrick96** [comment]
What version of polybar are you using? (`polybar -vv`)
Can you confirm that the include-file is loaded by running polybar with the `-l trace` argument (You may need to recompile and add `-DCMAKE_BUILD_TYPE=Debug ..` to the cmake command) and checking that you get a message along the lines of:
```
config: Including file "/home/myUser/.config/polybar/colors"
```

**Comment by Probotect0r** [comment]
I was using polybar v2.5, which I had installed via the polybar package on AUR. I was having trouble building (curl/curlbuild.sh file not found), so decided to try polybar-git package. Everything is good now. Although it would be nice to know why I couldn't build it from source. I saw the issue that was filed about the curlbuild.sh error, but that just said to get the latest version. I just cloned the repo, so code is latest. Should I be on a specific branch?

I was about to hit enter on the above message when I recalled seeing a branch option on the clone command under the compile instructions on the main page. Recloned without the option and was able to successfully.

**Comment by patrick96** [comment]
Either package on the AUR should correctly build (the `polybar` package includes a fix for the curlbuild issue inside the PKGBUILD).

If you manually clone this repo and build polybar, it should build the latest version. The instructions on the main page will build version 3.0.5 (the latest stable release). Maybe the repo that you originally had, was somehow stuck on 2.5, but cloning the repo again should always work, I've never had any problems with it.

**Comment by nathanchere** [comment]
To confirm- with trace enabled I definitely get the file included:

> * Loading config: /home/testymctestface/.config/polybar/config
> - config: Including file "/home/testymctestface/.config/polybar/.common.config"
> - config: Current bar section: [bar/top]

but then still get this:

> - module/memory: Deconstructing
> error: Disabling module "memory" (reason: The reference ${colors.black} does not exist (no fallback set))
>

**Comment by Probotect0r** [comment]
Can you try setting the black color to a static value instead of getting it
from xrdb?

**Comment by nathanchere** [comment]
I've tried that too. e.g. `black = #000000`.

I've also tried declaring a module in the included `.common.config` file then referencing it in a bar defined in the parent config. That works fine, so the include works in general. I can also reference the [colors] variables from the modules defined in that same file and included with it. Which makes me think I'm probably doing it wrong (e.g. do I need reference  included sections  differently?).
